### PR TITLE
Wd 559 i 2 c ingress module

### PR DESCRIPTION
**Pull request by GhasBarbouchi**

### CODE REVIEW

File: src/module/module.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from smbus2 import smbus2 as smbus
 
 MILL_PER_SEC = 1000
-I2Cbus = smbus.SMBus(int(getenv('I2C_INTERFACE_NUMBER')))
+I2Cbus = smbus.SMBus(int(getenv('I2C_INTERFACE')[-1]))
```
Comment (europaul) [inline src/module/module.py]: what if the interface number has more than one digit?

File: src/module/module.py [hunk 2]
```diff
@@ -20,8 +20,9 @@ def module_main():
     Implements module's main logic for inputting data.
     Function description should not be modified.
     """
-
+    
     log.debug("Inputting data...")
+    print(int(getenv('I2C_INTERFACE')[-1]))
```
Comment (europaul) [inline src/module/module.py]: is this a debug print? if so remove it

**Review by europaul — CHANGES_REQUESTED** [review]

File: src/module/module.py (hunk 2 above)
Comment (GhasBarbouchi) [reply src/module/module.py]: yes it is, removed in successive commits

File: src/module/module.py (hunk 1 above)
Comment (GhasBarbouchi) [reply src/module/module.py]: good question, but the problem is SMBus.SMBus(int) takes an int as a parameter and I have to confirm that for all the linux i2c interface editions, the number is put on the end.

Comment (europaul) [reply src/module/module.py]: no, I'm talking about one digit vs several digits. currently you are only extracting one digit

Comment (GhasBarbouchi) [reply src/module/module.py]: so weird to find any controller that has more than 10 (0 to 9) if that exists i2c interfaces many slaves possible for one interface).
I am with 1 digit.

File: src/module/module.py (hunk 2 above)
Comment (europaul) [reply src/module/module.py]: please remove such things prior to requesting a review

File: src/module/module.py (hunk 1 above)
Comment (europaul) [reply src/module/module.py]: please make it also accept multiple digits to make it more future proof

Comment (GhasBarbouchi) [reply src/module/module.py]: yes, that makes sense, I propose to use both the i2c interface path and the i2c interface number as inputs.

File: src/module/module.py (hunk 2 above)
Comment (GhasBarbouchi) [reply src/module/module.py]: sure

**Pull request closed without merging**